examples/mixture/plot_gmm.py

```diff
@@ -73,8 +73,9 @@ def plot_results(X, Y_, means, covariances, index, title):
           .7 * np.random.randn(n_samples, 2) + np.array([-6, 3])]
 
 # Fit a Gaussian mixture with EM using five components
-gmm = mixture.GaussianMixture(n_components=5, covariance_type='full').fit(X)
-plot_results(X, gmm.predict(X), gmm.means_, gmm.covariances_, 0,
+gmm = mixture.GaussianMixture(n_components=5, precision_type='full').fit(X)
```

Comment:
Just to be sure, we keep `precision_type` ?
For me, I prefer `precision_type` to be coherent with the attribute `precisions_`.